.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,5 @@
+Fix #[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issues #IssueNumber]
```

Comment:
This looks fine. But maybe you can add the extra instructions as HTML comments - 
```markdown
Fix #[Issue Number] <!-- If you do not solve the issue entirely, please change the message to something like "First steps for issue #[Issue Number] -->
```

Please let me know about your views.